src/controllers/mainController.ts

```diff
@@ -1205,6 +1215,126 @@ export default class MainController implements vscode.Disposable {
             ),
         );
 
+        // Start container command
+        this._context.subscriptions.push(
+            vscode.commands.registerCommand(
+                Constants.cmdStartContainer,
+                async (node: TreeNodeInfo) => {
+                    if (!node) return;
+                    try {
+                        // doing it this way instead of directly calling startContainer
+                        // allows for the object explorer item loading UI to show
+                        this._objectExplorerProvider.deleteChildrenCache(node);
+                        await this._objectExplorerProvider.setNodeLoading(node);
```

Comment:
NIT: Maybe this code should have been added in OE service so that we can keep track of everything in one place.